toplevel/coqloop.ml

```diff
@@ -450,24 +474,18 @@ let process_toplevel_command ~state stm =
     else
       let out = show_proof_diff_cmd ~state diff_opt in
       Feedback.msg_notice out;
-    state
-
-(* We return a new state and true if we got a `Drop` command  *)
-let read_and_execute_base ~state =
-  let input = top_buffer.tokens in
-  match read_sentence ~state input with
-  | Some G_toplevel.VernacDrop ->
-    if Mltop.is_ocaml_top()
```

Comment:
The reason to keep this separate was because in principle we could move `Drop.` to its own bytecode specific grammar file; not a big deal tho.